cmd/client/main.go

```diff
@@ -87,6 +89,16 @@ func withArgs(args []string, errorf errorFn) {
 	rootCmd.AddCommand(reportCmd(rootArgs, errorf))
 	rootCmd.AddCommand(quotaCmd(rootArgs, errorf))
 
+	// The tracer will keep track of spans in memory and print them to stdout when the client exits.
+	recorder := tracer.NewInMemoryRecorder()
```

Comment:
I think we want tracing to be controlled by a command-line flag, so by default this output is not produced.